src/api.js

```diff
@@ -0,0 +1,47 @@
+import BASE_URL from './constants';
+import { clearContent, notification } from './util.dom';
+
+import KEY from '../key'; // eslint-disable-line
```

Comment:
I am unable to run your app locally with `npm run dev` because the `key` module is missing.

Reply:
Hi @trekab, thanks for your review. Its explained on the readme. Kindly check the local setup instructions in the readme file.

Reply:
I missed it, sorry about that. Please resubmit this code review request.